packages/react-mutation-mapper/src/component/filter/BadgeSelector.tsx

```diff
@@ -18,6 +12,12 @@ import {
     handleOptionSelect,
 } from '../../util/SelectorUtils';
 import { BadgeLabel, DEFAULT_BADGE_STYLE } from './BadgeLabel';
+import {
+    CheckBoxType,
+    getSelectedValuesMap,
+    Option,
+} from '../../../../cbioportal-frontend-commons/src/components/checkedSelect/CheckedSelectUtils';
+import BadgeListSelector from '../../../../cbioportal-frontend-commons/src/components/checkedSelect/BadgeListSelector';
```

Comment:
This is wrong. We shouldn't import components from other packages this way. It should be like `import { BadgeListSelector } from 'cbioportal-frontend-commons'`